usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1690,38 +1774,122 @@
         ),
         "federal_account": OrderedDict(
             [
-                ("owning_agency_name", "treasury_account__federal_account__parent_toptier_agency__name"),
-                ("reporting_agency_name", "reporting_agency_name"),  # Column is annotated in account_download.py
+                ("owning_agency_name", "treasury_account__funding_toptier_agency__name"),
+                ("reporting_agency_name", "submission__reporting_agency_name"),
```

Comment:
Any particular reason to change this? Current functionality has this annotated with `StringAgg("submission__reporting_agency_name", "; ", distinct=True)` since Federal Account download will do a `GROUP BY`.

Reply:
I see what happened here. Reverted